cli/internal/codegen/client.go

```diff
@@ -54,7 +55,11 @@ func Client(lang Lang, appSlug string, md *meta.Data) (code []byte, err error) {
 	var gen generator
 	switch lang {
 	case LangTypeScript:
-		gen = &typescript{generatorVersion: typescriptGenLatestVersion}
+		if tsOptions == nil {
+			tsOptions = &daemonpb.GenClientRequest_TypeScriptOptions{Namespaces: true}
```

Comment:
Instead of doing this here we could make the client always pass in options as it's the thing that's aware of the presets, and the base case of "no preset" is essentially just the default preset